.github/workflows/docker.yml

```diff
@@ -0,0 +1,17 @@
+name: docker
+
+on:
+  push:
+    branches: main
+  pull_request:
+    branches: main
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
```

Comment:
The goal here is to build an image to test on your pc ? You are crushing the image used by the towers that must be built for arm. We have to setup a cross-build but we can't overwrite the existing image

Reply:
? I'm not sure what you mean: I haven't removed anything here, only added a test.

To me, we have to make sure that the image can at least be built on ubuntu. If it doesn't, we can expect it to fail on ARM. Now, I wanted to add also a CI to build the image on ARM but I have no clue how long it takes on RPI. I suggest to add this in another PR along with the orchestration of the CI to include API interactions :+1:

Reply:
You erase the [current image](https://hub.docker.com/repository/registry-1.docker.io/pyronear/pyro-engine/tags?page=1&ordering=last_updated) on hub but pushing a linux/arm64 image, but the docker pushis not here sorry !

Reply:
https://github.com/pyronear/pyro-engine/blob/ea017e1cff156f6d1137cd7684180288b7aa8a21/.github/workflows/release.yml#L58

Reply:
But yes we can fix it manually and managed this in a comming PR